setup.py

```diff
@@ -15,7 +15,8 @@
     long_description=long_description,
     keywords='taskotron fedora python rpm',
     author='Miro Hrončok, Iryna Shcherbina, Michal Cyprian',
-    author_email='mhroncok@redhat.com, ishcherb@redhat.com, mcyprian@redhat.com',
+    author_email=('mhroncok@redhat.com, ishcherb@redhat.com,'
+                  'mcyprian@redhat.com'),
```

Comment:
There's a missing space now and the string is: `'mhroncok@redhat.com, ishcherb@redhat.com,mcyprian@redhat.com'`